sunkit_magex/tests/__init__.py

```diff
@@ -1,4 +0,0 @@
-# Licensed under a 3-clause BSD style license - see LICENSE.rst
```

Comment:
I think we should keep this, because don't you need it to make `pytest --pyargs` work? Also this is a template decision.

Reply:
But its empty. It provides no tests.

Reply:
Also, is the CI not running the tests with pyargs?

Reply:
Don't we have init.py files in all the test dirs in core?

Reply:
They all have tests in them.